lib/index.js

```diff
@@ -194,7 +194,7 @@ exports.array = array;
  * if this is not enough, this can be increased.
  */
 function nonEmptyArray(speck) {
-    return createNonObjectSpeck(ioTsBrands_1.nonEmptyArray(speck._ioTsType), fc.array(speck._fastCheckArbitrary, 1, 5));
+    return createNonObjectSpeck((0, ioTsBrands_1.nonEmptyArray)(speck._ioTsType), fc.array(speck._fastCheckArbitrary, 1, 5));
```

Comment:
I've completely forgot, what does the (0,...) bit do?

Reply:
This is auto-generated. Honestly, who knows why it's there. `(0, x)` just evaluates to `x`

Reply:
Gives the code a bit of an edge